internal/component/faro/receiver/internal/payload/payload.go

```diff
@@ -238,6 +238,14 @@ func (m Measurement) KeyVal() *KeyVal {
 	}
 	MergeKeyVal(kv, m.Trace.KeyVal())
 	MergeKeyValWithPrefix(kv, KeyValFromMap(m.Context), "context_")
+
+	valuesAsString := make(map[string]string, len(m.Values))
+	for key, value := range m.Values {
+		valuesAsString[key] = fmt.Sprintf("%v", value)
+	}
+
+	MergeKeyValWithPrefix(kv, KeyValFromMap(valuesAsString), "value_")
```

Comment:
If I'm reading this right, this adds these values as strings but we continue storing them as floats as well under the original key?

Reply:
Yes exactly. 
I kept the original values to not introduce a breaking change.